mkosi/machine.py

```diff
@@ -48,9 +48,10 @@ def flush(self) -> None:
 
 
 class Machine:
-    def __init__(self, args: Sequence[str] = []) -> None:
+    def __init__(self, args: Sequence[str] = [], retry_amount: int = 30) -> None:
```

Comment:
```suggestion
    def __init__(self, args: Sequence[str] = [], *, retry_amount: int = 30) -> None:
```
I'd prefer to make keyword arguments with defaults to only be callable via keyword

Reply:
Let's rename it to `num_retries` as well

Reply:
Yep, via keyword seems better. Regarding your other comment, I didn't quite understand what "plumb through" the `num_retries` to pytest would be, so I tried to implement what I thought correct.

Let me know if I got it wrong.

Reply:
Sorry for being unclear: What I meant is to be able to run pytest such, that i can set `num_retries` on the CLI.